SETUP.md

```diff
@@ -20,21 +16,24 @@ type in `"Uniswap"` or `"Uniswap V3"`, for the Sushi Swap protocol you may type
 4. For the `protocolAbbreviation` key, type in an appropriate abbreviation for the value in `protocolName`.  For
 example, `"Uniswap"` may be abbreviated `"UNI"` and `"Sushi Swap"` may be abbreviated `"SUSH"`, etc.
 
-5.  The Object value for the `contractName1` key corresponds to addresses that we want to monitor.  Each
-key in the Object is an address (either EOA or contract), and each value is another object with three fields:
-  -`name`: the name of the contract or EOA that will be watched
-  -`type`: the type of finding that will be generated when transactions involving this address are detected (see
-  Forta SDK for `Finding` types)
-  -`severity`: the severity of the finding that will be generated when transactions involving this address are
-  detected (see Forta SDK for `Finding` severities)
+5. Check out the SETUP.md in each agent directory for details on configuring each agent you want to use
+
+6. Set up any abi files that your agents need in the following directory structure:
+```
+  abi/
+    <agent_name>/
+      <abi_file.json>
+    <agent_name>/
+      <abi_file.json>
+```
 
 6. Create a new README.md file to provide a description of your agent, using examples from the Forta Github
-repository.  Additionally, update the `name` entry in `package.json` to match the values provided in the 
+repository.  Additionally, update the `name` entry in `package.json` to match the values provided in the
 `agent-config.json` file.
 
 7. Move files to have the following directory structure:
   ```
-  address-watch/
+  forta-agent-templates/
```

Comment:
Should this sample directory structure be revised more? We need to indicate where the needed template code goes (i.e. under `/src/<template-name>`).